src/android/io/branch/BranchSDK.java

```diff
@@ -466,12 +466,12 @@ private void createBranchUniversalObject(JSONObject options, CallbackContext cal
      * @param options A {@link JSONObject} value to set URL options.
      * @param controlParams A {@link JSONObject} value to set the URL control parameters.
      */
-    private void showShareSheet(int instanceIdx, JSONObject options, JSONObject controlParams) throws JSONException
+    private void showShareSheet(int instanceIdx, JSONObject options, JSONObject controlParams, String shareText) throws JSONException
     {
 
         Log.d(LCAT, "start showShareSheet()");
 
-        ShareSheetStyle shareSheetStyle = new ShareSheetStyle(this.activity, "Check this out!", "This stuff is awesome: ")
+        ShareSheetStyle shareSheetStyle = new ShareSheetStyle(this.activity, "Check this out!", shareText)
```

Comment:
Yes, I supposed, however iOS doesn't have a functionality to modify the share title text according to @jestoniyap . So atm I disregarded the modification of the title, though I can still add the functionality for Android but not on iOS. If it's okay with you.